onnxruntime/python/tools/tensorrt/perf/build/build_image.py

```diff
@@ -17,8 +17,8 @@
 TRT_DOCKER_FILES = {
     "8.6.cuda_11_8_cudnn_8": "tools/ci_build/github/linux/docker/Dockerfile.ubuntu_cuda11_8_tensorrt8_6",
     "8.6.cuda_12_3_cudnn_9": "tools/ci_build/github/linux/docker/Dockerfile.ubuntu_cuda12_3_tensorrt8_6",
-    "10.3.cuda_11_8_cudnn_8": "tools/ci_build/github/linux/docker/Dockerfile.ubuntu_cuda11_tensorrt10",
-    "10.3.cuda_12_5_cudnn_9": "tools/ci_build/github/linux/docker/Dockerfile.ubuntu_cuda12_tensorrt10",
+    "10.4.cuda_11_8_cudnn_8": "tools/ci_build/github/linux/docker/Dockerfile.ubuntu_cuda11_tensorrt10",
```

Comment:
Is there plan to decouple cuda and cudnn version so that we can use latest cudnn version (like 9.3) with cuda 11.8?

Reply:
> @onnxruntime-es,
> 
> Is there plan to decouple cuda and cudnn version so that we can use latest cudnn version (like 9.3) with cuda 11.8?

probably this can be discussed separately from this TRT 10.4 update.
my preference is to limit the number of combinations.  what is pytorch behavior currently? if you install cuda 11.8 version, does it depend on cudnn9 or 8? 
how long will we continue to support cuda 11.8 ? if not much longer, it may not be worth changing cudnn version.  
either way, i think we'd want to encourage users to move to cuda 12.x (if limiting cudnn9 support to cuda 12.x only is one way of achieving that, maybe it's a good thing)

Reply:
I agree that cudnn version change is independent of TRT 10.4 update.

Pytorch use cudnn 9 for cuda 11.8:
https://github.com/pytorch/pytorch/blob/ec25ca453703a4e9fd3dae170ac2daa891427f13/.ci/docker/common/install_cudnn.sh#L9-L10

Reply:
ok i see. we probably should match them and drop cudnn8 support?